include/appimage/appimage_legacy.h

```diff
@@ -10,23 +10,6 @@
  * All of the functions in this header are deprecated and must not be used in newly written code
  */
 
-/*
```

Comment:
I don't see a reason for this change. Please only include changes that relates to the topic.